### DISCUSSION TITLE
How do I know it's select mode when the selectkey code is pressed?

### DISCUSSION THREAD

**Discussion by kyh0703** [xyflow/xyflow#4075] [Q&A] [answered]
Hi guys!
I want to change cursor when the selection key pressed.
But i don't know..
Plz help me..

**Comment by moklick** [comment, marked as answer]
Does this work for you?

```js
const isSelectionKeyPressed = useKeyPress('Shift');
```

https://reactflow.dev/api-reference/hooks/use-key-press

**Reply by kyh0703** [reply]
Thank you! I'll try🙏
